modules/python/nitf/source/nitro.i

```diff
@@ -156,6 +156,17 @@
     }
 }
 
+%typemap(in) nrt_IOHandle{
+    if ($input)
+    {
+        $1 = (nrt_IOHandle) PyInt_AsLong($input);
+    }
+}
+
+%typemap(out) nrt_IOHandle{
+    $result = SWIG_From_int((int)($1));
```

Comment:
Similarly here... I'm not familiar with this syntax but do you need `PyInt_FromSize_t()`?